petab/petablint.py

```diff
@@ -49,34 +49,35 @@ def parse_cli_args():
     )
 
     # Call with set of files
-    parser.add_argument(
+    group = parser.add_argument_group("Check individual files *DEPRECATED*")
```

Comment:
Could we add to this message, so that if someone happens to rely on individual files, we are informed?

```suggestion
    group = parser.add_argument_group("Check individual files *DEPRECATED*. Please contact us via GitHub issues, if you need this.")
```